docs/tutorials/boot-tutorial.md

```diff
@@ -144,43 +161,68 @@ Now, to build the disk image, inside the disk-image folder, run:
 ./packer build boot-exit/boot-exit.json
 ```
 
+Once this process succeeds, the disk image can be found on `boot-exit/boot-exit-image/boot-exit`.
+A disk image already created following the above instructions can be found, gzipped, [here](http://dist.gem5.org/dist/v20-1/images/x86/ubuntu-18-04/boot-exit.img.gz).
+
+
 ## Compiling the linux kernel
 
-In this tutorial, we want to experiment with different linux kernels to examine the state of gem5's ability to boot different linux kernels. We picked one of the latest stable kernels (v5.2.3) and the last four LTS (long term support) releases which include: v4.19.83, v4.14.134, v4.9.186, and v4.4.186.
+In this tutorial, we want to experiment with different linux kernels to examine the state of gem5's ability to boot different linux kernels. These tests use following five LTS (long term support) releases of the Linux kernel:
+
+- 4.4.186
+- 4.9.186
+- 4.14.134
+- 4.19.83
+- 5.4.49
 
-Let's use an example of kernel v5.2.3 to see how to compile the kernel.
+
+Let's use an example of kernel v5.4.49 to see how to compile the kernel.
 First, add a folder linux-configs to store linux kernel config files. The configuration files of interest are available [here](https://github.com/darchr/gem5art/blob/master/docs/linux-configs/).
```

Comment:
Did you mean to update this with a gem5-resources pointer?

Reply:
I have updated the link to gem5-resources linux configs. I will remove linux-configs folder from gem5art/docs once other tutorials are updated as well.